Update Permanent_license.md (#3674)
SkylineCommunications/dataminer-docs: user-guide/Reference/DataMiner_Licenses/Permanent_license.md

   At this point, startup will fail.

> 1. Provide Dataminer Licensing Group (dataminer.licensing@skyline.be) with a copy of the file *request.lic*, located in the folder *C:\\Skyline DataMiner\\*.

Reviewer: ```suggestion
1. Email a copy of the file *request.lic*, located in the folder *C:\\Skyline DataMiner\\*, to <dataminer.licensing@skyline.be>.
```
